tests/system/common.py

```diff
@@ -83,3 +84,22 @@ def wait_for_mesos_endpoint(timeout_sec=timedelta(minutes=5).total_seconds()):
     it returns false"""
 
     return shakedown.time_wait(lambda: shakedown.mesos_available_predicate(), timeout_seconds=timeout_sec)
+
+
+def metronome_available_predicate():
+    url = metronome_api_url()
+    try:
+        response = http.get(url)
+        return response.status_code == 200
+    except Exception as e:
+        return False
+
+def metronome_api_url():
+    return shakedown.dcos_url_path("/service/metronome/v1/jobs")
```

Comment:
`shakedown.dcos_service_url("metronome/v1/jobs")` ?  not a big deal

Reply:
that does not work :( because what that method does is: "/service/metronome/v1/jobs/" - it puts slash after the last word which is not what I want :(